_query-dsl/compound/function-score.md

```diff
@@ -826,4 +826,198 @@ The results contain the three matching blog posts:
   }
 }
 ```
-</details>
\ No newline at end of file
+</details>
+
+## Named functions
+
+When defining a function, you can specify its name using the `_name` parameter at the top level. This name is useful for debugging and understanding the scoring process. Once specified, the function name is included in the score calculation explanation whenever possible (this applies to functions, filters, and queries). You can identify the function by its `_name` in the response.
+
+### Example
+
+The following request sets `explain` to `true` for debugging purposes in order to obtain scoring explanation in the response. Each function contains a `_name` parameter so you can identify the function unambiguously:
```

Comment:
```suggestion
The following request sets `explain` to `true` for debugging purposes in order to obtain a scoring explanation in the response. Each function contains a `_name` parameter so that you can identify the function unambiguously:
```